src/index.html

```diff
@@ -1,13 +1,298 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
+
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
+    <title>BOSE</title>
+    <link rel="icon" href="//static.bose.com/etc/designs/bose/consumer-products-2016/favicon.ico">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;700;800&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
     <script src="scripts/main.js" defer></script>
   </head>
-  <body>
-    <h1>Miami</h1>
+
+  <body class="page__body">
+    <header class="header page__section">
+      <div class="container">
+        <div class="header__content">
+          <div class="top-actions">
+            <a
+              href="https://www.bose.com/en_us/index.html"
+              class="logo"
+              target="_blank"
+            >
+              <img class="logo__image" src="./images/photos/image 4.png" alt="BOSE">
```

Comment:
3 і більше атрибути - перенесимо на новий рядок кожен атрибут
```suggestion
              <img 
                class="logo__image" 
                src="./images/photos/image 4.png" 
                alt="BOSE"
              >
```

Reply:
Відформатуй правильно всі зображення